src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -207,11 +222,23 @@ export class CalcitePickListItem {
     );
   }
 
+  renderRemoveAction() {
+    if (!this.removable) {
+      return null;
+    }
+    return (
+      <calcite-action class={CSS.remove} text="remove" onClick={this.removeClickHandler}>
+        <calcite-icon scale="s" icon="x" />
+      </calcite-action>
+    );
+  }
+
   renderSecondaryAction() {
     const hasSecondaryAction = this.el.querySelector(`[slot=${SLOTS.secondaryAction}]`);
-    return hasSecondaryAction ? (
+    return hasSecondaryAction || this.removable ? (
       <div class={CSS.action}>
         <slot name={SLOTS.secondaryAction} />
+        {this.renderRemoveAction()}
```

Comment:
Nitpick: Would moving this into the slot as default content help against users providing a secondary action + removable?

Reply:
not sure I follow. You mean add this manually to the lightDOM instead of to the shadow dom? Editing the light dom seems risky as we might accidentally override user defined slotted content.